src/main/java/org/jitsi/videobridge/health/Health.java

```diff
@@ -311,26 +230,11 @@ protected void doRun()
      * of {@code videobridge} or the check determines that {@code videobridge}
      * is not healthy.
      */
-    public void check()
+    @Override
+    protected void performCheck()
         throws Exception
     {
-        Exception lastResult = this.lastResult;
-        long lastResultMs = this.lastResultMs;
-        long timeSinceLastResult = System.currentTimeMillis() - lastResultMs;
-
-        if (timeSinceLastResult > Config.getTimeout())
-        {
-            throw new Exception(
-                "No health checks performed recently, the last result was "
-                    + timeSinceLastResult + "ms ago.");
-        }
-
-        if (lastResult != null)
-        {
-            throw new Exception(lastResult);
-        }
-
-        // We've had a recent result, and it is successful (no exception).
+        doCheck(videobridge);
```

Comment:
I'm too lazy to expand the whole file in the diff...does `doCheck` get called from multiple places? Could we just put the logic here?